engine/flink/components/table/src/main/scala/pl/touk/nussknacker/engine/flink/table/source/FlinkMiniClusterTableOperations.scala

```diff
@@ -113,37 +129,25 @@ object FlinkMiniClusterTableOperations extends LazyLogging {
         .schema(flinkTableSchema)
         .build()
     )
-    tempDir
+    tempDir -> tableName
   }
 
-  private def cleanup(dir: Path, tableNames: List[String])(implicit env: TableEnvironment): Unit = {
-    def delete(dir: Path): Unit = Try {
-      Files
-        .walk(dir)
-        .sorted(java.util.Comparator.reverseOrder())
-        .forEach(path => Files.deleteIfExists(path))
-      logger.debug(s"Successfully deleted temporary test data dumping directory at: '${dir.toUri.toURL}'")
-    } match {
-      case Failure(e) =>
-        logger.error(
-          s"Couldn't properly delete temporary test data dumping directory at: '${dir.toUri.toURL}'",
-          e
-        )
-      case Success(_) => ()
-    }
-    def deleteTable(tableName: String)(implicit env: TableEnvironment): Unit = {
```

Comment:
until we dont have a single persistent minicluster per designer, we don't need to delete tables